recipes/r-rcppplanc/meta.yaml

```diff
@@ -0,0 +1,118 @@
+{% set version = '1.0.0-rc2' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-rcppplanc
+  version: {{ version|replace("-", "_") }}
+
+
+source:
+  url: https://github.com/welch-lab/RcppPlanc/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 20663306ba3f52073a33d435e14015d3667f3b266d820379364b53cce0606f20
+
+build:
+  skip: True  # [win]
+  # If this is a new build for the same version, increment the build number.
+  number: 0
+
+  # This is required to make R link correctly on Linux.
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+# Suggests: knitr, withr, rmarkdown, testthat (>= 3.0.0)
+requirements:
+  build:
+    - {{ compiler('c') }}              # [not win]
+    - {{ compiler('m2w64_c') }}        # [win]
+    - {{ compiler('cxx') }}            # [not win]
+    - {{ compiler('m2w64_cxx') }}      # [win]
+    - {{ posix }}filesystem        # [win]
+    - {{ posix }}git
+    - {{ posix }}zip               # [win]
+    - {{ stdlib("c") }}
+    - cmake >=3.22
+    - make
+    - llvm-openmp  # [osx]
+    - libgomp      # [linux]
+    - pkg-config
+    - diffutils  # [not win]
+    - m2-diffutils  # [win]
+    - patch  # [not win]
+    - m2-patch  # [win]
+
+  host:
+    - r-base
+    - r-matrix
+    - r-rcpp
+    - r-rcpparmadillo
+    - r-rcppprogress
+    - libhwloc >=2.0.0,<3.0.0
+    - hdf5 >=1.10
+    - libcblas
+    - liblapack
+
+  run:
+    - r-base
+    - {{ native }}gcc-libs         # [win]
+    - r-matrix
+    - r-rcpp
+    - r-rcpparmadillo
+    - r-rcppprogress
+    - libblas  # [not (osx and arm64)]
+    - libblas * *accelerate  # [osx and arm64]
```

Comment:
Drop:

```suggestion
```

Adding `libblas` in **host** will resolve the errors. Generally, it doesn't make sense to restrict the runtime BLAS/LAPACK unless one actually builds against specific versions and takes advantage of specific implementations.

Reply:
This is intentional-accelerate uses the inbuilt  matrix co-processor on ARM macs and the performance is significantly better than openmp w/ any other blas impl.